multipaz-zkp/src/commonMain/kotlin/org/multipaz/zkp/ZkSpec.kt

```diff
@@ -0,0 +1,21 @@
+package org.multipaz.zkp
+
+/**
+ * Kotlin representation of the native ZkSpecStruct
+ */
+class ZkSpec(
+    // The ZK system name and version
+    val system: String,
+    // The hash of the compressed circuit
+    val circuitHash: String,
+    // The number of attributes that the circuit supports
+    val numAttributes: Long,
+    // The version of the ZK specification
+    val version: Long
+) {
+
+    override fun toString(): String {
+        return "ZkSpec(system='$system', circuitHash='$circuitHash', " +
+                "numAttributes=$numAttributes, version=$version)"
+    }
+}
```

Comment:
I think this should be a data class, that way I think the built-in `toString()` method might also be good enough and do close to what the override here is.

Also, need proper kdoc, please use `@property` or `@param` inside the class kdoc block to document the parameters. Ditto elsewhere, all public API should be well-documented.